pkg/cmd/pipeline/start.go

```diff
@@ -376,6 +381,10 @@ func getOptionsByType(resources resourceOptionsFilter, restype string) []string
 
 func (opt *startOptions) startPipeline(pName string) error {
 	pr := &v1alpha1.PipelineRun{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "tekton.dev/v1alpha1",
```

Comment:
How to specify the APIVersion seems difficult. This could maybe be pulled from the pipeline?

Reply:
I think we would want at least to have it in a global since that string is scattered everywhere, but that's a refactoring that can be done for later....

Reply:
:thinking: I think it's fine for now, but indeed, good question. I would guess we default to a hard coded version, and we could have some smart code that checks what api version are available in the cluster (and adjust from it or error out from there)